R/utils.R

```diff
@@ -21,3 +21,17 @@ epidist_version_stanvar <- function() {
   comment <- paste0("// code chunks used from epidist ", version, "\n")
   brms::stanvar(scode = comment, block = "functions")
 }
+
+#' Round to the nearest multiple
+#'
+#' This function rounds an input `x` down to the nearest multiple of some number
+#' `f`. For example, if `f = 0.2` and `x = 1.5` then the output would be 1.4.
+#' Or, if `f = 1` then `floor_mult` behaves as `floor`.
+#'
+#' @param x A number to be rounded down
+#' @param f A number specifying the multiple to be rounded down to
+#' @family utils
+#' @export
```

Comment:
do we want to export this? Arg for is useful for people to. have to construct there own thing. Arg against is that is mainly developers and its more stuff to support

Reply:
Am fine with not exporting it, will make that change